### DISCUSSION TITLE
请教下，如何给两个节点画2条线，两条线单独设置标签和箭头？跪求方案，在线等。

### DISCUSSION THREAD

**Discussion by zhaoweikai** [antvis/X6#1889] [Q&A] [answered]
类似下图中的红框，给两个节点创建两个线。
![image](https://user-images.githubusercontent.com/10459842/157867967-16e83b7d-6fc2-4cd9-bfd7-7d23f4b517ae.png)

**Comment by NewByVector** [comment, marked as answer]
默认支持多条线的，你的问题是不是怎么让两条线分开？如果是的话，可以使用 [nodeanchor](https://x6.antv.vision/zh/docs/api/registry/node-anchor/#gatsby-focus-wrapper) 来设置偏移

**Reply by zhaoweikai** [reply]
非常感谢，这个就可以解决。唉，项目太着急了，没时间好好把所有文档看完。
